2023/28xxx/CVE-2023-28426.json

```diff
@@ -5,87 +5,14 @@
     "CVE_data_meta": {
```

Comment:
('Expecting property name enclosed in double quotes: line 18 column 1 (char 625)',)